src/GameManipulator.php

```diff
@@ -227,15 +249,16 @@ public function didShootHitOpponentShip(Shoot $shoot, Game $game): ?Ship
      * 
      * @param Ship $ship
      * @param Game $game
+     * @param Player $player
      * 
      * @return bool
      */
-    public function isShipSunk(Ship $ship, Game $game): bool
+    public function isShipSunk(Ship $ship, Game $game, Player $player): bool
```

Comment:
Better! But i'm still confused about which player it is? The current one? The opponent?